script/Arcueid/Day 5/05_00_ARC05_2_2.txt

```diff
@@ -96,7 +96,7 @@ No matter what I think, this is not an opponent that I can handle.
 [sha:a129c00d9d96d0105207ba1af15ff5e1249d9676]{
 -- Page 209, Offset 9109.
 -- だが、それで<お|・><ま|・><え|・>は正しいのか。
-But is this really the right decision?
+But is this really the right %{i}decision%{/i}?
```

Comment:
Doesn't really work because the JP here is actually
> However, is that correct for _you_?

with the sentence as is, I would italicize 
> But is this _really_ the right decision?